package.json

```diff
@@ -14,12 +14,14 @@
     "@fortawesome/react-fontawesome": "^0.1.14",
     "bootstrap": "^4.5.3",
     "d3": "3",
+    "formik": "^2.2.6",
     "jquery": "^3.5.1",
     "lodash": "^4.17.20",
     "next": "^10.0.7",
     "popper.js": "^1.16.1",
     "react": "16.14.0",
-    "react-dom": "16.14.0"
+    "react-dom": "16.14.0",
+    "yarn": "^1.22.10"
```

Comment:
Oops, I don't think `yarn` should be listed here (I think generally it should be installed globally). Just pushed a commit (0a8c80c62a21d70b339af71ef27b63fd84ca497d) to remove it from the dependencies.

Reply:
Thanks!